views/partials/searchCard.handlebars

```diff
@@ -0,0 +1,25 @@
+<div class="search-card">
+  <div class="search-left">
+    <div class="search-dropoff-info">
+      <h1 class="search-name">{{title}}</h1>
+      <h2 class="search-address">{{address}}</h2>
+      <h2 class="search-phone">{{phone}}</h2>
+    </div>
+    <div class="search-dropoff-options">
+      <p>Drop off 
+        {{#if dropoff}} <span class="check">✓</span> {{else}} <span class="cross">✗</span> {{/if}}
+      </p>
+      <p>Mail in 
+        {{#if mailin}} <span class="check">✓</span> {{else}} <span class="cross">✗</span> {{/if}}
+      </p>
+    </div>
+  </div>
+  <div class="search-right">
+    <h2>Items</h2>
```

Comment:
nit: If you want you can change the "Items" term to "Items Accepted" or "Accepts" to convey thats the item the organization accepts.